ivadomed/scripts/convert_to_onnx.py

```diff
@@ -1,17 +1,21 @@
 import argparse
 import torch
 
-from ivadomed import utils as imed_utils
+from ivadomed.utils import init_ivadomed, Metavar, save_onnx_model
 
 
 def get_parser():
     parser = argparse.ArgumentParser()
-    parser.add_argument("-m", "--model", dest="model", required=True, type=str, help="Path to .pt model.")
+    parser.add_argument("-m", "--model", dest="model", required=True, type=str,
+                        help="Path to .pt model.", metavar=Metavar.str)
```

Comment:
```suggestion
                        help="Path to .pt model.", metavar=Metavar.file)
```
The path should point to a ".pt" file.

Reply:
I'll change that one :)

Reply:
Fixed here: https://github.com/ivadomed/ivadomed/pull/641/commits/bc1691bdb29c57f758096efd0731c57e21580d82